src/editor/logic.py

```diff
@@ -282,7 +282,7 @@ def send_editorial_review_update(book, review_assignment, email_text, sender, at
         'sender': sender,
```

Comment:
These long lines upset me too but changing these would mean changing everything in that document which would also upset me.